### PR TITLE
Billboards Ownership Style Rules

### PR DESCRIPTION
**Pull request by lboogie04**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
 If the owner of the billboard is an organization, style it with the organization flair(billboard display_ad with an `organization_id`)
 
Screenshot 2023-02-28 at 10 26 50 PM
Screenshot 2023-02-28 at 10 26 59 PM

## Related Tickets & Documents

- Related Issue #
- Closes [#532](https://github.com/forem/forem-internal-eng/issues/532)

## QA Instructions, Screenshots, Recordings

1. Create `billboards` slug in admin on Pages view
2. Create display ads with an `organization_id`
3. View the billboard display ad with the organization logo and org name

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

- [ ] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Review by jaw6 — APPROVED** [review]
Looks good to me, but @anujbhavsar96 should have a look I think.

File: app/views/shared/_display_ad.html.erb [hunk 1]
```diff
@@ -4,7 +4,12 @@
   data-category-impression="<%= DisplayAdEvent::CATEGORY_IMPRESSION %>"
   data-context-type="<%= data_context_type %>">
   <div class="crayons-sponsorship__header relative">
-    <div class="crayons-sponsorship__title"><%= Settings::Community.community_name %></div>
+    <% if Page.exists?(slug: "billboards") && display_ad.organization_id? && display_ad.type_of == "community" %>
+      <img width="24" height="24" class="radius-default crayons-sponsorship__image" src="<%= display_ad.organization.profile_image_url_for(length: 64) %>" alt="profile" loading="lazy" />
+      <div class="crayons-sponsorship__title pl-2 fs-s"><%= display_ad.organization.name %></div>
```
Comment (rt4914) [inline app/views/shared/_display_ad.html.erb:9]: Change pl-2 to pl-1 so that the gap between text/org-name and icon is 8px. Currently its 12px

**Review by rt4914 — APPROVED** [review]
One minor nit suggestion.

**Review by anujbhavsar96 — APPROVED** [review]
Same suggestion as Rajat about spacing. 

Apart from it, looks good to me 🚀

File: app/views/shared/_display_ad.html.erb [hunk 2]
```diff
@@ -4,7 +4,12 @@
   data-category-impression="<%= DisplayAdEvent::CATEGORY_IMPRESSION %>"
   data-context-type="<%= data_context_type %>">
   <div class="crayons-sponsorship__header relative">
-    <div class="crayons-sponsorship__title"><%= Settings::Community.community_name %></div>
+    <% if Page.exists?(slug: "billboards") && display_ad.organization_id? && display_ad.type_of == "community" %>
```
Comment (benhalpern) [inline app/views/shared/_display_ad.html.erb:7]: Is `Page.exists?(slug: "billboards")` necessary in this context? It seems to add an extra query we probably don't need for this logic, eh?

**Pull request merged**